samples/dashboards/demo/climate/demo.js

```diff
@@ -684,14 +684,20 @@ async function setupBoard() {
         });
     }
 
-    // Load initial city
-    await setupCity(board, activeCity, activeColumn, activeScale);
+    // Load cities
+    for (let i = 0, iEnd = cityRows.length; i < iEnd; ++i) {
```

Comment:
Previously we had some sort of a `lazy loading` but right now we wait for each city.
I would revert this change to have a better initial performance.

Reply:
Is it safe to call updateBoards() before all cities are loaded? Placing updateBoards() before or after city loading has an effect on the initial temperatures shown.

Reply:
Hmm, the `updateBoard` should not affect the initial temperatures because we already should have the initial data.
With the current state everything works for me as expected.

Reply:
This is strange: On my setup, I observe that following (https://www.highcharts.com/demo/dashboards/climate):

1. After loading the temperature is, say, Dublin 12, Bangkok 25.
2. When click on any map marker and the temperatures of all but New York have changed.

I see the same behaviour on Windows 11(Chrome, Edge) and Ubuntu 20.4 (Chrome, Firefox).